src/server/routes/enrichment/index.js

```diff
@@ -211,281 +211,21 @@ enrichmentRouter.post('/visualization', (req, res) => {
- *           - agambiae
- *           - aglabripennis
- *           - amellifera
- *           - apisum
- *           - aqueenslandica
- *           - avaga
- *           - bantarctica
- *           - bimpatiens
- *           - bmalayi
- *           - bmori
- *           - cbrenneri
- *           - cbriggsae
- *           - celegans
- *           - cgigas
- *           - cjaponica
- *           - cquinquefasciatus
- *           - cremanei
- *           - cteleta
- *           - dananassae
- *           - derecta
- *           - dgrimshawi
- *           - dmelanogaster
- *           - dmojavensis
- *           - dpersimilis
- *           - dplexippus
- *           - dponderosae
- *           - dpseudoobscura
- *           - dpulex
- *           - dsechellia
- *           - dsimulans
- *           - dvirilis
- *           - dwillistoni
- *           - dyakuba
- *           - hmelpomene
- *           - hrobusta
- *           - iscapularis
- *           - lanatina
- *           - lcuprina
- *           - lgigantea
- *           - lloa
- *           - lsalmonis
- *           - mcinxia
- *           - mdestructor
- *           - mleidyi
- *           - mscalaris
- *           - nvectensis
- *           - nvitripennis
- *           - obimaculoides
- *           - ovolvulus
- *           - phumanus
- *           - ppacificus
- *           - rprolixus
- *           - sinvicta
- *           - smansoni
- *           - smaritima
- *           - smimosarum
- *           - spurpuratus
- *           - sratti
- *           - sscabiei
- *           - tadhaerens
- *           - tcastaneum
- *           - tkitauei
- *           - tspiralis
- *           - turticae
- *           - znevadensis
- *           - alyrata
- *           - atauschii
- *           - athaliana
- *           - atrichopoda
- *           - bdistachyon
- *           - bnapus
- *           - boleracea
- *           - brapa
- *           - bvulgaris
- *           - ccrispus
- *           - cmerolae
- *           - creinhardtii
- *           - gmax
- *           - gsulphuraria
- *           - hvulgare
- *           - lperrieri
- *           - macuminata
- *           - mtruncatula
- *           - obarthii
- *           - obrachyantha
- *           - oglaberrima
- *           - oglumaepatula
- *           - oindica
- *           - olongistaminata
- *           - olucimarinus
- *           - omeridionalis
- *           - onivara
- *           - opunctata
- *           - orufipogon
- *           - osativa
- *           - ppatens
- *           - ppersica
- *           - ptrichocarpa
- *           - sbicolor
- *           - sitalica
- *           - slycopersicum
- *           - smoellendorffii
- *           - stuberosum
- *           - taestivum
- *           - tcacao
- *           - tpratense
- *           - turartu
- *           - vvinifera
- *           - zmays
+ *           - "ENSEMBL"
+ *           - "HGNC"
+ *           - "HGNCSYMBOL"
+ *           - "UNIPROT"
+ *           - "NCBIGENE"
```

Comment:
we don't need to document species anymore?

Reply:
I think we're only supporting human, since PC is generally human-only.

Reply:
On line 214 for `query` I say your input has to be human. 
Also, g:Convert is tricky as it subsets the `target` database field based on species, so I just removed it.